src/main/java/com/epam/edumanagementsystem/util/service/UserService.java

```diff
@@ -13,4 +13,6 @@ public interface UserService {
     List<User> findAll();
 
     User findByEmail(String email);
+
+    Boolean checkDuplicationOfEmail(String email);
```

Comment:
For boolean methods it is prefered to check names in if/else context. In this case if (emailExists()).....